int8 arrow_0
int8 arrow_1
int8 yaw
int8 grip